django/contrib/admin/checks.py

```diff
@@ -28,6 +29,18 @@ def _issubclass(cls, classinfo):
         return False
 
 
+def _module_exists(dotted_path, paths):
+    """
+    Check if dotted module path is found among paths. Also takes care to
+    ensure any superclass of the queried path is found.
+    """
+    middleware_cls = import_string(dotted_path)
```

Comment:
`middleware` might not be the best variable name since it's used for authentications backend as well. What about 

```suggestion
    subclass = import_string(subclass_path)
```

Reply:
Ah, yes – silly me, I initially solved it specifically for the middleware case, but then generalized it later without taking care to rename my variables :man_shrugging: Thanks!